src/modules/service-catalog/components/service-catalog-list-item/LoadingState.tsx

```diff
@@ -0,0 +1,64 @@
+import { Col, Grid, Row } from "@zendeskgarden/react-grid";
+import { Skeleton } from "@zendeskgarden/react-loaders";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+import { MD, XXXL } from "@zendeskgarden/react-typography";
+import styled from "styled-components";
+
+const SkeletonItem = styled(Col)`
```

Comment:
At smaller viewports, when the items are on multiple lines, there is no vertical space between them

Reply:
Right, I added `margin-bottom` for small viewports

Reply:
I just noticed that this component is a styled `Col` and it is not needed since is already wrapped in a `Col` component, we can change it to a styled `div`